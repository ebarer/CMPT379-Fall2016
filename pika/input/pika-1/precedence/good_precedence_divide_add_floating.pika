exec {

  print 9.5 / 6.8 + 2.7, _n_.
  print 8.0 / 2.2 + 4.4, _n_.
  print 7.7 / 4.4 + 2.6, _n_.
  print 9.7 / 5.4 + 3.5, _n_.
  print 7.9 / 6.4 + 9.4, _n_.
  print 5.8 / 5.9 + 9.6, _n_.
  print 7.2 / 3.3 + 9.7, _n_.
  print 4.0 / 6.0 + 8.1, _n_.
  print 6.4 / 2.3 + 4.3, _n_.
  print 8.0 / 7.8 + 3.3, _n_.
  print 4.3 / 5.7 + 8.0, _n_.
  print 1.7 / 7.8 + 8.5, _n_.

  print 5.8 + 7.8 / 8.1, _n_.
  print 5.5 + 5.9 / 4.3, _n_.
  print 9.4 + 9.7 / 6.1, _n_.
  print 9.0 + 3.7 / 1.3, _n_.
  print 9.8 + 9.0 / 9.6, _n_.
  print 6.8 + 1.2 / 1.6, _n_.
  print 9.4 + 1.0 / 6.3, _n_.
  print 5.9 + 4.1 / 8.4, _n_.
  print 7.2 + 3.1 / 7.3, _n_.
  print 9.9 + 6.1 / 4.8, _n_.
  print 5.5 + 2.1 / 3.5, _n_.
  print 9.1 + 4.2 / 9.4, _n_.

  print 5.0 / 3.1 + 3.9 / 8.9, _n_.
  print 1.3 / 9.7 + 8.1 / 9.9, _n_.
  print 7.9 / 2.2 + 4.1 / 7.7, _n_.
  print 7.6 / 9.5 + 2.0 / 9.0, _n_.
  print 7.6 / 6.6 + 4.8 / 5.4, _n_.
  print 8.6 / 6.5 + 9.2 / 6.6, _n_.
  print 6.2 / 1.2 + 9.4 / 9.4, _n_.
  print 7.4 / 3.3 + 6.0 / 9.6, _n_.
  print 7.7 / 8.9 + 5.9 / 2.3, _n_.
  print 7.9 / 1.1 + 6.8 / 5.8, _n_.
  print 8.0 / 8.5 + 6.9 / 8.4, _n_.
  print 3.3 / 8.3 + 8.4 / 9.3, _n_.

  print 9.7 + 6.6 / 7.1 + 3.4, _n_.
  print 8.2 + 7.7 / 1.3 + 9.9, _n_.
  print 2.6 + 6.2 / 7.7 + 2.4, _n_.
  print 5.3 + 3.4 / 6.9 + 4.5, _n_.
  print 6.2 + 8.3 / 8.5 + 9.5, _n_.
  print 4.1 + 8.0 / 9.0 + 9.5, _n_.
  print 6.1 + 1.0 / 5.2 + 6.9, _n_.
  print 5.2 + 4.2 / 7.9 + 6.8, _n_.
  print 4.3 + 9.3 / 8.6 + 7.2, _n_.
  print 2.6 + 9.3 / 1.0 + 6.1, _n_.
  print 6.1 + 5.7 / 9.4 + 7.1, _n_.
  print 6.1 + 7.2 / 9.4 + 2.9, _n_.

}

