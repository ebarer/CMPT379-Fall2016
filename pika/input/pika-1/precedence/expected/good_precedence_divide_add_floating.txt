4.09706
8.03636
4.35
5.2963
10.6344
10.5831
11.8818
8.76667
7.08261
4.32564
8.75439
8.71795
6.76296
6.87209
10.9902
11.8462
10.7375
7.55
9.55873
6.3881
7.62466
11.1708
6.1
9.54681
2.05111
0.952202
4.12338
1.02222
2.0404
2.71702
6.16667
2.86742
3.43039
8.35423
1.76261
1.30082
14.0296
24.0231
5.80519
10.2928
16.6765
14.4889
13.1923
12.5316
12.5814
18
13.8064
9.76596
